### ISSUE TITLE
Pretrained weights don't give same accuracy as stated in repo

### ISSUE THREAD

**Issue by neiladit** [lukemelas/EfficientNet-PyTorch#231]
I tied using pretrained weights for efficientnet-b0 and efficientnet-b1 and evaluated it on Imagenet but did not get the same Accuracy (rank 1 and rank 5) that is mentioned in the table [here](https://github.com/lukemelas/EfficientNet-PyTorch#usage). I'm using the evaluation code from the [repository itself](https://github.com/lukemelas/EfficientNet-PyTorch/blob/master/examples/imagenet/main.py). Here is what I got:

efficientnet-b0: `Prec@1` **74.802** `Prec@5` **91.998**
efficientnet-b1: `Prec@1` **74.638** `Prec@5` **91.674**

However, some of them gave similar results to the README table:
efficientnet-b2: `Prec@1` **79.408** `Prec@5` **94.604**
efficientnet-b3: `Prec@1` **81.196** `Prec@5` **95.530 **

Can you verify this and let me know if there is an issue?

**Comment by ooodragon94** [comment]
hi mate
EfficientNet is very sensitive to resolution
if you have seen the paper, you can see that what they are optimizing is compound of depth width and resolution of the network.
Therefore, you MUST upscale the given image to specified image scale

I've been there, haha
and got same accuracy as yours

**Comment by neiladit** [comment]
@ooodragon94 Thanks for the comment. Doesn't [this line](https://github.com/lukemelas/EfficientNet-PyTorch/blob/5fbffa4461e0f0e944da6b1dc1c2e1441b108d0d/examples/imagenet/main.py#L218) deal with that?

**Comment by neiladit** [comment]
Since I'm using the file from the repository, I don't think anything is different. Were you able to replicate the results?

**Comment by ooodragon94** [comment]
I was
It was quite long ago though; I manually had to upscale images to get the specified results

**Comment by neiladit** [comment]
@lukemelas can you confirm if there is a modification required in the [Imagenet evaluation code](https://github.com/lukemelas/EfficientNet-PyTorch/blob/5fbffa4461e0f0e944da6b1dc1c2e1441b108d0d/examples/imagenet/main.py)? Thanks a lot!

**Comment by derekwan1** [comment]
Got the exact same numbers; any update on this?
